cmd/chainbridge/main.go

```diff
@@ -53,12 +53,12 @@ var accountCommand = cli.Command{
 	Name:     "accounts",
 	Usage:    "manage bridge keystore",
 	Category: "KEYSTORE",
```

Comment:
should we change the category of this as well?

Reply:
Not a bad idea, removing it altogether would be good

Reply:
Sure I can remove it. Does category signify anything or is it just for readability?

Reply:
Yeah, its a built in feature of the CLI library, doesn't do much for us. An example usage is the `docker` command (has `Commands` and `Management Commands`).

Reply:
@luu-alex  Please can you remove all usages of the category field in this file